bin/setup.sh

```diff
@@ -26,21 +26,14 @@ export COOPR_RUBY=${COOPR_RUBY:-${COOPR_HOME}/provisioner/embedded/bin/ruby}
 
 export COOPR_PROVISIONER_PLUGIN_DIR=${COOPR_HOME}/provisioner/lib/worker/plugins
 
-if [ -r "${TRUST_CERT_PATH}" ] && [ -n "${TRUST_CERT_PASSWORD}" ]; then
-  export CERT_PARAMETER="--cert ${TRUST_CERT_PATH}:${TRUST_CERT_PASSWORD}"
-fi
-
-if [ "${COOPR_PROTOCOL}" == "https" ]; then
-  export CURL_PARAMETER="--insecure"
-fi
-
 wait_for_plugin_registration () {
   RETRIES=0
-  until [[ $(curl ${CURL_PARAMETER} --silent --request GET \
```

Comment:
I understand why we are removing the if statements, and other code with CERT_PARAMETER.  Why are we removing the ones with "CURL_PARAMETER"?